schema/src/main/jade-tables/library_prep_activity.table.json

```diff
@@ -6,35 +6,13 @@
       "datatype": "string",
       "type": "primary_key"
     },
-    {
-      "name": "cross_references",
-      "datatype": "string",
-      "type": "repeated"
-    },
-    {
-      "name": "time_created",
```

Comment:
`time_created` and `cross_references` could both be pulled from the raw library, but I don't see much of a point in duplicating the info across tables.

Reply:
That makes sense to me.